chain/consensus/filcns/filecoin.go

```diff
@@ -369,7 +369,8 @@ func (filec *FilecoinEC) VerifyWinningPoStProof(ctx context.Context, nv network.
 		}
 	}
 
-	ok, err := ffiwrapper.ProofVerifier.VerifyWinningPoSt(ctx, proof.WinningPoStVerifyInfo{
+	// TODO(rvagg): why is this using proofsffi.ProofVerifier.VerifyWinningPoSt directly and not filec.verifier.VerifyWinningPoSt?
```

Comment:
Worth capturing a github issue for this for triage?

Reply:
Yeah, thanks for spotting, I forgot about this but I wrote it when I understood the codebase a lot less than I do now. So here's a PR to fix it, which we can land later: https://github.com/filecoin-project/lotus/pull/12358